test/spec/features/rules/BpmnRulesSpec.js

```diff
@@ -58,7 +58,8 @@ describe('features/modeling/rules - BpmnRules', function() {
           sequenceFlow = elementFactory.createConnection({
```

Comment:
As we're fixing something I'd expect a test case that verifies the new behavior.